mdpow/tests/test_workflows_base.py

```diff
@@ -7,42 +7,64 @@
 
 import pybol
 import pytest
+import numpy as np
+from numpy.testing import assert_equal
 import pandas as pd
+import MDAnalysis as mda
 
 from . import RESOURCES, MANIFEST, STATES
 from pkg_resources import resource_filename
 from mdpow.workflows import base
 
-@pytest.fixture(scope='function')
+@pytest.fixture()
 def molname_workflows_directory(tmp_path):
     m = pybol.Manifest(str(MANIFEST))
     m.assemble('workflows', tmp_path)
     return tmp_path
 
 class TestWorkflowsBase(object):
 
-    @pytest.fixture(scope='function')
+    reference_guessed_elements = np.array(['C', 'N', 'C', 'Cl', 'C'])
```

Comment:
The test failure is due to capitalization:
```suggestion
    reference_guessed_elements = np.array(['C', 'N', 'C', 'CL', 'C'])
```